sample/QueueTrigger-Powershell/function.json

```diff
@@ -1,17 +1,19 @@
-﻿{
-    "bindings": [
-        {
-            "type": "queueTrigger",
-            "direction": "in",
-            "queueName": "samples-powershell"
-        },
-        {
-            "type": "table",
-            "name": "output",
-            "direction": "out",
-            "tableName": "samples",
-            "partitionKey": "samples-powershell",
-            "rowKey": "{id}"
-        }
-    ]
-}
+﻿{
+  "bindings": [

```

Comment:
I already did the cleanup pass on all files in this repo. They're all 4 spaces which you convinced me to move to.